src/app/debug/table/table.component.html

```diff
@@ -37,24 +37,28 @@
       </select>
     </div>
 
-    <div class="btn btn-static my-2 mx-1 px-2"> Reports in progress: {{ tableSettings.reportsInProgress }}</div>
-    <div class="row" *ngIf="tableSettings.reportsInProgress > 0">
+    <div class="btn btn-static my-2 mx-1 px-2"> Reports in progress: {{ tableSettings.numberOfReportsInProgress }}</div>
+    <div class="row" *ngIf="tableSettings.numberOfReportsInProgress > 0">
       <div class="btn btn-static my-2 mx-1 px-2"> Estimated memory usage of reports in progress: {{ tableSettings.estimatedMemoryUsage }} Bytes</div>
         <div class="col input-group my-2 mx-1 px-2">
-          <input #openIndex type="number" min="0" value="{{tableSettings.reportsInProgress}}" max="{{tableSettings.reportsInProgress}}" (change)="disableReportInProgressButton(+openIndex.value, '#openReportInProgressButton')" class="form-control" data-cy-debug="openInProgressNo"/>
+          <input #openIndex type="number" min="0" value="{{tableSettings.numberOfReportsInProgress}}" max="{{tableSettings.numberOfReportsInProgress}}" (change)="disableReportInProgressButton(+openIndex.value, '#openReportInProgressButton')" class="form-control" data-cy-debug="openInProgressNo"/>
           <div class="input-group-append">
             <button data-cy-debug="openInProgress" class="btn btn-info" type="button" id="openReportInProgressButton" (click)="openReportInProgress(+openIndex.value)">Open</button>
           </div>
         </div>
           <div class="col input-group my-2 mx-1 px-2">
-            <input #deleteIndex type="number"  min="0" value="{{tableSettings.reportsInProgress}}" max="{{tableSettings.reportsInProgress}}" (change)="disableReportInProgressButton(+openIndex.value, '#deleteReportInProgressButton')" class="form-control" id="deleteInProgressNo"/>
+            <input #deleteIndex type="number" min="0" value="{{tableSettings.numberOfReportsInProgress}}" max="{{tableSettings.numberOfReportsInProgress}}" (change)="disableReportInProgressButton(+openIndex.value, '#deleteReportInProgressButton')" class="form-control" id="deleteInProgressNo"/>
```

Comment:
Why do `max="{{tableSettings.numberOfReportsInProgress}}"` when you can make this a binding? Same goes for value
```
[value]="tableSettings.numberOfReportsInProgress" [max]=tableSettings.numberOfReportsInProgress
```

Always try to not interpolate what should be a binding.
Do pay attention to value since you need to use `[ngValue]` if its an object, but this is most likely only needed for `<select>` and its `<option>`s